pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dao/kit/KitDao.java

```
@@ -186,6 +190,30 @@ public Boolean isBloodKit(String ddpLabel) {
         return booleanCheckFoundAsName(ddpLabel, SQL_IS_BLOOD_KIT_QUERY);
     }
 
+    public String getCollaboratorParticipantId(String ddpParticipantId) {
+        if (StringUtils.isBlank(ddpParticipantId)) {
+            return null;
+        }
+
+        SimpleResult results = inTransaction(conn -> {
+            SimpleResult dbVals = new SimpleResult("");
+            try (PreparedStatement stmt = conn.prepareStatement(SQL_GET_COLLABORATOR_PARTICIPANT_ID)) {
+                stmt.setString(1, ddpParticipantId);
+                try (ResultSet rs = stmt.executeQuery()) {
+                    if (rs.next()) {
+                        dbVals.resultValue = rs.getString(DBConstants.COLLABORATOR_PARTICIPANT_ID);
+                    }
+                }
+            } catch (SQLException ex) {
+                dbVals.resultException = new DsmInternalError("Error getting collaborator participant Id for participant: " + ddpParticipantId, dbVals.resultException);
```

This code effectively swallows this exception.
I suggest either throwing an exception, or logging a ERROR and returning a value the caller can use, perhaps NULL as above.